GNNGraphs/src/temporalsnapshotsgnngraph.jl

```diff
@@ -54,10 +54,10 @@ GNNGraph:
 ```
 """
 struct TemporalSnapshotsGNNGraph{G<:GNNGraph, D<:DataStore}
-    num_nodes::Vector{Int}   
-    num_edges::Vector{Int}
+    num_nodes::AbstractVector{Int}   
+    num_edges::AbstractVector{Int}
     num_snapshots::Int
-    snapshots::Vector{G}
+    snapshots::AbstractVector{G}
     tgdata::D   
 end
```

Comment:
these changes shouldn't be needed because of this
https://github.com/JuliaGraphs/GraphNeuralNetworks.jl/pull/573/files#diff-1aa1e7fe1f4b8ca3c78522bb95d8600bab501a27d9f49bf9fe56df14aaa901aaR65

try after a rebase